File: VexTel.App/ClientApp/src/app/models/simulacaoChamada.ts
import { DDD } from "./ddd";
import { Plano } from "./plano";

export class SimulacaoChamada {
  id: number;
  dddOrigemId: number;
  dddDestinoId: number;
  planoId: number;
  tempo: number;
  custoComFaleMais: string;
  custoSemFaleMais: string;
  dddOrigem: DDD;
  dddDestino: DDD;
  plano: Plano;
}

File: VexTel.Domain/Entities/SimulacaoChamada.cs
namespace VexTel.Domain.Entities
{
    public class SimulacaoChamada
    {
        public int Id { get; set; }
        public int DDDOrigemId { get; set; }
        public virtual DDD DDDOrigem { get; set; }
        public int DDDDestinoId { get; set; }
        public virtual DDD DDDDestino { get; set; }
        public int PlanoId { get; set; }
        public virtual Plano Plano { get; set; }
        public int Tempo { get; set; }
        public string CustoComFaleMais { get; set; }
        public string CustoSemFaleMais { get; set; }   

    }
}
